python/lsst/drp/tasks/assemble_cell_coadd.py

```diff
@@ -221,8 +257,9 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
             skyMap, tractId=outputDataId["tract"], patchId=outputDataId["patch"]
         )
 
+        units = CoaddUnits.legacy if self.config.do_scale_zero_point else CoaddUnits.nJy
```

Comment:
The value of `do_scale_zero_point` is essentially used to identify whether the warps are in units of `nJy` or not. Given that we haven't used this for any serious purpose yet, we _could_ drop support to make cell-coadds from old warps but I have retained that capability now although in a less robust manner.